roles/unicorn/tasks/main.yml

```diff
@@ -10,14 +10,14 @@
     group={{ item.group }}
   become: yes
   with_items:
+    # Any updates to the unicorn init script need to be reloaded: systemctl daemon-reload
     - { src: "unicorn_init.j2", dest: "/etc/init.d/unicorn_{{ app }}", mode: "0744", owner: "{{ unicorn_user }}", group: "{{ unicorn_user }}" }
     - { src: "unicorn.rb.j2", dest: "{{ unicorn_config }}", mode: "0744", owner: "{{ unicorn_user }}", group: "{{ unicorn_user }}" }
   notify:
     - restart nginx
 
 - name: enable unicorn at boot
   service:
-    daemon_reload=yes # Ensure unicorn init script is reloaded
     name=unicorn_{{ app }}
```

Comment:
I think that option wouldn't have hurt but it wasn't very clear there either. Since enabling the service happens after the init scripts have been copied, a reload would have happened afterwards before enabling the service (or before checking that it's enabled). Anyway, we don't need to invest more time here.